infra/testing/key.go

```diff
@@ -0,0 +1,6 @@
+package testing
+
+import "github.com/CoreumFoundation/coreum/pkg/types"
+
+// HornOfPlentyPrivKey is used by testing framework to fund accounts required by integration tests
+var HornOfPlentyPrivKey = types.Secp256k1PrivateKey{0x86, 0x2f, 0x17, 0xc3, 0x51, 0x83, 0xbe, 0x2a, 0x2b, 0x5d, 0x5b, 0x5b, 0xb5, 0x53, 0x86, 0xd8, 0xad, 0xc8, 0xde, 0x51, 0xa9, 0x73, 0x3f, 0xb7, 0x7d, 0x72, 0xb9, 0x29, 0x91, 0xb7, 0x2c, 0x60}
```

Comment:
I would recommend to keep the mnemonic or hex key as a string here, sine it's more readble and in the future in will be easier to set it as a ENV variable